debian/postinst

```diff
@@ -28,7 +30,16 @@ rsync -a /var/archivematica/storage-service/templates /usr/lib/archivematica/sto
 cd /usr/lib/archivematica/storage-service
 
 echo "configuring django database and static files"
-/usr/share/python/archivematica-storage-service/bin/python manage.py syncdb
+find /usr/lib/archivematica/storage-service/ -name '*.pyc'  -delete
+# 0.3.0 will automatically fake the first migration
+if [[ $2 == '0.6.'* || $2 == '0.5.'* || $2 == '0.4.'* ]]; then
+    echo 'Faking the 0.4 migration'
+    /usr/share/python/archivematica-storage-service/bin/python manage.py migrate locations 0002_v0_4 --fake
+fi
+if [[ $2 == '0.6.'* || $2 == '0.5.'* ]]; then
```

Comment:
Check with Justin that these actually match the versions postinst will be called with.